ledger/Cargo.toml

```diff
@@ -37,6 +37,9 @@ rand = { workspace = true }
 rand_chacha = { workspace = true }
 rayon = { workspace = true }
 reed-solomon-erasure = { workspace = true, features = ["simd-accel"] }
+rocksdb = { workspace = true, features = ["lz4"] }
+# Avoid the vendored bzip2 within rocksdb-sys that can cause linker conflicts
+ # when also using the bzip2 crate
```

Comment:
Please keep the comment above the line that the comment is referring to. Also, please avoid the leading whitespace in the second line of the comment